<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositCalc</class>
 <widget class="QWidget" name="DepositCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DepositCalc</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="SumLabel">
        <property name="text">
         <string>Сумма вклада</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_sum">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="RubLabel">
        <property name="text">
         <string>₽</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="timeLabel">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Срок размещения</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_time">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButto_mounth">
        <property name="text">
         <string>Месяц</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_year">
        <property name="text">
         <string>Год</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="perLabel">
        <property name="text">
         <string>Процентная ставка</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_per">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="percentLab">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="perioLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Периодичность выплат</string>
        </property>
       </widget>
       <widget class="QComboBox" name="periodBox">
        <property name="minimumSize">
         <size>
          <width>222</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>222</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Ежемесячно</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ежеквартально</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ежегодно</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QCheckBox" name="capitalizationCheck">
       <property name="text">
        <string>Капитализация процентов</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="deposit">
          <property name="text">
           <string>Переодичные пополнения</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_monthly_deposit">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="RubLabel_2">
         <property name="text">
          <string>₽</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLabel" name="withdrawl">
          <property name="text">
           <string>Переодичные снятия</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_monthly_withdrawal">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="RubLabel_3">
         <property name="text">
          <string>₽</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_calc">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 10px;
  background-color: rgb(55, 55, 255);
  color: white; 
  border: 2px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="perL">
            <property name="text">
             <string>Начисленные проценты</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="taxL">
            <property name="text">
             <string>Налог</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sumL">
            <property name="text">
             <string>Сумма к концу срока</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="1">
           <widget class="QLabel" name="sumRL">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>₽</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="taxRL">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>₽</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="perRL">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>₽</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="sum_">
            <property name="maximumSize">
             <size>
              <width>291</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tax_">
            <property name="maximumSize">
             <size>
              <width>291</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="per_">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>291</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_back">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  border-radius: 10px;
  background-color: rgb(255, 51, 57);
  color: white; 
  border: 2px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
        </property>
        <property name="text">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
